internal/accounts/accountstest.go

```diff
@@ -3,6 +3,7 @@ package accounts
 // Copyright (C) 2023 by Posit Software, PBC.
```

Comment:
I think it is probably a mistake that this file was named without an underscore between accounts and test. I know this was ahead of your changes, but do you mind changing it? I don't think it gets executed as part of the test suite, if it isn't named correctly.